ibm_mq/datadog_checks/ibm_mq/connection.py

```diff
@@ -56,3 +55,13 @@ def get_ssl_connection(config):
     queue_manager.connect_with_options(config.queue_manager_name, cd, sco)
 
     return queue_manager
+
+
+def _get_channel_definition(config):
+    cd = pymqi.CD()
+    cd.ChannelName = pymqi.ensure_bytes(config.channel)
+    cd.ConnectionName = pymqi.ensure_bytes(config.host_and_port)
+    cd.ChannelType = pymqi.CMQC.MQCHT_CLNTCONN
+    cd.TransportType = pymqi.CMQC.MQXPT_TCP
+    cd.Version = config.mqcd_version
```

Comment:
Is there any reason to default to `mqcd_version: 6` instead of not setting the `Version` and let the library deal with it ?

```suggestion
    if config.mqcd_version is not None:
        cd.Version = config.mqcd_version
```

And

`mqcd_version = instance.get('mqcd_version', 6)` => `mqcd_version = instance.get('mqcd_version', None)`


The library default to `6`, but this might change in the future: https://github.com/dsuch/pymqi/blob/d013276894c30f7ceb11e243763dbacff076d51b/code/pymqi/__init__.py#L827

Reply:
Enforcing the default to be 6 in our side allows us to document what the default is